src/main/java/baritone/utils/PathRenderer.java

```diff
@@ -267,28 +266,7 @@ private static void drawGoal(@Nullable BufferBuilder bufferBuilder, PoseStack st
                 //      inject hook into LevelRenderer#renderBlockEntities where the matrices have already been set up correctly
                 //      copy out and modify the vanilla beacon render code
                 //  also another issue on 1.21.5 is we don't have a simple method call for editing the beacon's depth test
-
-                stack.pushPose(); // push
-                stack.translate(goalPos.getX() - renderPosX, -renderPosY, goalPos.getZ() - renderPosZ); // translate
-
-                BeaconRenderer.renderBeaconBeam(
```

Comment:
i think its reasonable to give up on this lol 😂 does anyone use it?

Reply:
hasn't worked since at least MC 1.20.4 iirc

and i haven't heard anyone complain

/shrug